package.json

```diff
@@ -19,8 +19,8 @@
     "coverage": "istanbul cover _mocha -- -t 20000 test/index.js",
     "coveralls": "npm run coverage && coveralls < coverage/lcov.info",
     "standard": "standard",
-    "test": "mocha --reporter list -t 20000 test/index.js",
-    "bundle-test": "browserify test/index.js > test/bundle.js"
+    "test": "mocha --reporter spec -t 20000 test/index.js",
+    "test:bundle": "browserify test/index.js >test/browser.js"
```

Comment:
I think it's more logical, it's test related task, but honestly I don't care about names. Maybe I should put `karma`?